### PR TITLE
Update changes from initial

### PR DESCRIPTION
**Pull request by deep-quality-dev**
* refactor: propose transaction onto Gnosis Safe
* fix: use Azure endpoint instead of Gnosis Safe API
* feat: add listAssetsCoins and listAssetsCollectibles
* feat: use Azure endpoint for listing transactions

### CODE REVIEW

File: src/client/GnosisSafeClient.ts [hunk 1]
```diff
@@ -145,16 +103,9 @@ class GnosisSafeClient {
...
-      address,
-      selectedCurrency,
-      {
-        exclude_spam: true,
-        trusted: false,
-      }
-    );
+    const url = `https://zero-service-gateway.azure-api.net/gnosis/${network}/safes/${address}/balances/${selectedCurrency}?exclude_spam=true&trusted=false`;
+    const res = await fetch(url);
+    return await res.json();
```
Comment (Remscar) [inline src/client/GnosisSafeClient.ts:108]: Can we wrap this (and any other api call we make here) in a try/catch and fall back to the gnosis safe sdk method if the zero-service-gateway fails?

Comment (deep-quality-dev) [reply src/client/GnosisSafeClient.ts:108]: Good point, thanks, I will fix it

**Review by Remscar — APPROVED** [review]
This PR is a bit too large for me to fully review. Please try and break these changes up into smaller PR's in the future

**Comment by Remscar** [comment]
Also the Circle CI pipeline is failing, please refresh the yarn lock file

**Pull request merged**